static/app/views/alerts/utils/getMetricRuleDiscoverUrl.spec.tsx

```diff
@@ -18,7 +19,7 @@ describe('getMetricRuleDiscoverQuery', () => {
         end: new Date().toISOString(),
       },
     });
-    expect(query.valueOf()).toEqual(
+    expect(query!.valueOf()).toEqual(
```

Comment:
is there another way we can use to avoid the usage of `!`?

Reply:
this is related to returning null https://github.com/getsentry/sentry/pull/53486/files#diff-9348321af70420e95f7338168b9405e6b38774ba27eeb82fbfc5f0b1b2139b97L51 before we were abusing that string happens to also have a valueOf function